Gems/GraphCanvas/Code/Source/Components/Slots/Data/DataSlotLayoutComponent.cpp

```diff
@@ -392,7 +400,14 @@ namespace GraphCanvas
 
     void DataSlotLayout::OnNameChanged(const AZStd::string& name)
     {
-        m_slotText->SetLabel(name);
+        if (m_isNameHidden)
+        {
+            m_slotText->SetLabel("");
+        }
+        else
+        {
+            m_slotText->SetLabel(name);
+        }
```

Comment:
Looks like a place you can use a ternary operator. (Also 
```suggestion
        m_slotText->SetLabel(m_isNameHidden?"":name);
```